<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitReportReporter -->
<testsuite hostname="DESKTOP-G6NDRKF" failures="0" tests="2" name="com.fithub.services.chat.test.suites.ChatroomServiceTest" time="1.623" errors="0" timestamp="2024-04-04T13:56:46 CEST" skipped="0">
  <testcase classname="com.fithub.services.chat.test.suites.ChatroomServiceTest" name="testCreateNewChatroom_InvalidDataIsProvided_ThrowsException" time="0.038"/>
  <testcase classname="com.fithub.services.chat.test.suites.ChatroomServiceTest" name="testCreateNewChatroom_ChatroomForCoachAlreadyExists_ThrowsBadRequestException" time="1.585"/>
</testsuite> <!-- com.fithub.services.chat.test.suites.ChatroomServiceTest -->
